guides/doc-Administering_Red_Hat_Satellite/topics/Disabling_Weak_Encryption.adoc

```diff
@@ -1,151 +1,41 @@
-[[Disabling_Weak_SSL_Encryption]]
-
-== Disabling Weak Encryption
-
-You might want to change the encryption settings for {Project} depending on the security requirements of your infrastructure or to fix vulnerabilities quickly. Use the following sections to disable weak SSL encryption and 64-bit cipher suites.
-
 [[Disabling_Weak_SSL_2-0_and_3-0_Encryption]]
-=== Disabling Weak SSL 2.0 and SSL 3.0 Encryption
+== Disabling TLS 1.0 and TLS 1.1 Encryption
 
-If your {Project} fails Nessus scans because of SSL vulnerabilities, or your security infrastructure requires that you disable SSL 2.0 and SSL 3.0, you can edit the `/etc/foreman-installer/custom-hiera.yaml` file to remove weak encryption.
+You might want to change the encryption settings for {Project} depending on the security requirements of your infrastructure or to fix vulnerabilities quickly.
 
-.Disabling Weak SSL 2.0 and SSL 3.0 Encryption for {Project}
+Apache and Qpid services in {Project} have TLS 1.0 and 1.1 encryption enabled by default. Use this procedure on {Project} and {SmartProxy} to configure {Project} and {SmartProxy} to only allow TLS 1.2, as required, for example, by PCI-DSS regulations.
 
-To disable weak encryption for {Project}, complete the following steps:
+.Procedure
 
 . Open the `/etc/foreman-installer/custom-hiera.yaml` file for editing:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # vi /etc/foreman-installer/custom-hiera.yaml
 ----
-+
-. Add the following entries:
+
+. Add the following entry:
 +
 [options="nowrap"]
 ----
-# Foreman Proxy
-foreman_proxy::tls_disabled_versions: [ '1.1' ]
-
-# Dynflow
-foreman_proxy::plugin::dynflow::tls_disabled_versions: [ '1.1' ]
-
 # Apache
 apache::mod::ssl::ssl_protocol: [ 'ALL' , '-SSLv3' , '-TLSv1' , '-TLSv1.1' , '+TLSv1.2' ]
 
-# Tomcat / Candlepin
-candlepin::tls_versions: [ '1.2' ]
-
 # QPID Dispatch
-foreman_proxy_content::qpid_router_ssl_protocols: [ 'TLSv1.2' ]
 foreman_proxy_content::qpid_router_ssl_ciphers: 'ALL:!aNULL:+HIGH:-SSLv3:!IDEA-CBC-SHA'
 ----
-+
-. Rerun the `{foreman-installer}` tool with no arguments:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {foreman-installer}
-----
-+
-. Restart the `{foreman-maintain}` services:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {foreman-maintain} service restart
-----
-
-
-.Disabling Weak SSL 2.0 and SSL 3.0 Encryption for {SmartProxy}
 
-To disable weak encryption for {SmartProxy}, complete the following steps:
-
-. Open the `/etc/foreman-installer/custom-hiera.yaml` file for editing:
+. Enter the `{foreman-installer}` command to configure Qpid to use TLS 1.2 only:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# vi /etc/foreman-installer/custom-hiera.yaml
-----
-+
-. Add the following entries:
-+
-[options="nowrap"]
+# {foreman-installer} \
+--foreman-proxy-content-qpid-router-ssl-protocols=TLSv1.2
```

Comment:
So I noticed that when the ciphers were tightened it also stopped listening on TLS 1.0 and 1.1 which would make that redundant. Should we drop this line?

Reply:
Do you say that we do not need to run the installer at all and that the config lines in custom-hiera.yaml would do the trick? If so, I can remove this step completely and return the step that restarts the services.

Reply:
No, still run the installer but without the argument. Running the installer is crucial to actually making the config file changes.

Reply:
Did you notice that I removed the `foreman_proxy_content::qpid_router_ssl_protocols: [ 'TLSv1.2' ]` config line? Now we say to only set the `foreman_proxy_content::qpid_router_ssl_ciphers: 'ALL:!aNULL:+HIGH:-SSLv3:!IDEA-CBC-SHA'` configuration. Will that do the trick? Want to check before committing it.

Reply:
It appeared to be when I tried it out. Now I'm not sure if it was just nmap's reporting. Will try to figure it out.

If you're unsure, you can do both for now and we can simplify it in a future iteration.